core/src/refx/effects.cljc

```diff
@@ -19,7 +20,9 @@
 
 (defn- db-effect [db]
   (when-not (identical? @app-db db)
-    (reset! app-db db)))
+    (subs/start-batch-update!)
+    (reset! app-db db)
+    (subs/end-batch-update!)))
```

Comment:
I'd prefer it if we kept the `db` namespace agnostic to `subs`. Maybe we can move this to the signal implementation for Atoms in `refx.subs`?

This would also allow to include state from other atoms in the batch update.